src/js/actions.js

```diff
@@ -177,7 +164,7 @@ export function submitForm(formConfig, form) {
         } else if (errorMessage.startsWith('vets_server_error')) {
           errorType = 'serverError';
         }
-        captureError(error, errorType);
+        recordEvent({ event: 'form-submit-error', error, errorType });
```

Comment:
These changes have to occur in a major version release so they're breaking changes regardless. Most likely the only client using this logic at the moment is vets.gov because of the very specific configuration and messages (e.g., `vets_server_error` above). In their `recordEvent` setup they can just map `form-submit-error` to `submission-failed` if they want to keep the same name. I just wanted to make the names consistent and since these three were about the form submit process I used the same prefix for their names.